web/src/copy_and_paste.ts

```diff
@@ -433,6 +433,31 @@ export function paste_handler_converter(paste_html: string): string {
             return "~~" + content + "~~";
         },
     });
+    turndownService.addRule("latexMath", {
+        filter(node: Node) {
+            if (node instanceof Element) {
+                const closestDisplay = node.closest(".katex-display");
+                return closestDisplay !== null;
+            }
```

Comment:
I feel like the `filter` should check for the `katex-mathml` being actually present inside?